components/automate-chef-io/layouts/_default/data-api.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>ReDoc</title>
+    <title>Chef Automate documentation</title>
```

Comment:
should this be `Chef Automate API Documentation`?

Reply:
All of the text in here should be regarded as barely informed placeholders :)

This came from me wondering where the page title came from, initially suspecting it was from metadata, but it turned out to be a plain old html title. I can update this.